src/components/newsletter/preview.js

```diff
@@ -0,0 +1,11 @@
+import parse from 'html-react-parser';
+
+export default function HTMLPreview(props){
+    return(
+        <html>
+            {
+                 parse(props.html)
+             }
+        </html>
+    )
+}
```

Comment:
Intended for previewing the newsletter before sending :)

Reply:
should work with simple texts as the body :)

Reply:
But again, this feature wasn't in the project description. The backend and the front end for the API request and the preview button were completed. Just some minor bug fixes and this feature can be implemented